tests/ImageSharp.Tests/Formats/Jpg/JpegColorConverterTests.cs

```diff
@@ -20,7 +20,7 @@ public class JpegColorConverterTests
 
     private const int TestBufferLength = 40;
 
-    private const HwIntrinsics IntrinsicsConfig = HwIntrinsics.AllowAll | HwIntrinsics.DisableAVX;
+    private const HwIntrinsics IntrinsicsConfig = HwIntrinsics.AllowAll | HwIntrinsics.DisableHWIntrinsic;
```

Comment:
I wonder why don't these tests fail with `JpegColorConverterArm64`?

Reply:
Me too, we need to re-review the tests, they should fail.

Reply:
@stefannikolei or @brianpopow  since you have a HW - can you try injecting an exception into the `CmykArm64` methods and running `dotnet test -f net6.0 --filter FullyQualifiedName~JpegColorConverterTests`? If it gets hit, there's probably a bug in the tests since it they didn't show the regression, if not, it's probably a bug in `FeatureTestRunner`.

Reply:
those failed: 
ImageSharp\tests\ImageSharp.Tests\Formats\Jpg\JpegColorConverterTests.cs:line 81

But I also saw all xxxFromVector methods fail because it is not supported on the hardware(vm)

Reply:
@antonfirsov The `FeatureTestRunner` seems to work as expected. Some path of the intrinsics methods (`ConvertFromRgb`) were not covered by tests. That's why the error was not detected. I have added tests for those.

Reply:
I have decided to revert the `IntrinsicsConfig` back to `HwIntrinsics.AllowAll | HwIntrinsics.DisableAVX2`. I think it was a mistake to change it. The `IntrinsicsConfig` is only used by the Vector color conversion tests. On x64 disabling AVX2 makes sense to test SSE here.

This means on ARM the test will be run twice, once 'HwIntrinsics.AllowAll` and once `HwIntrinsics.DisableAVX2'. I dont know howto change it so the second run will be avoided, but I also think its not a big deal. If anyone has a better idea let me know.

Reply:
Perhaps we can use this lib https://github.com/AArnott/Xunit.SkippableFact and just skip the not supported runs. 

So we can define the desired ones (avx/sse/arm) as a theory.

Reply:
I dont think its really worth the extra effort only to skip an addional test run on ARM. We maybe need to re-evaluate once we add more ARM code / tests.